src/screens/ProfileScreen/index.js

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import ImagePicker from 'react-native-image-crop-picker';
+import PropTypes from 'prop-types';
 import Toast from 'react-native-simple-toast';
+import netInfo from '@react-native-community/netinfo';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` has been removed but it's not clear if it was being used in the file. If it was, this could cause issues. Please ensure that `PropTypes` is not required anywhere in the code.

```diff
- import PropTypes from 'prop-types';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->